### ISSUE TITLE
Search field in a List view

### ISSUE THREAD

**Issue by Tacioandrade** [saltcorn/saltcorn#1613]
Guys, the application went into production on Monday and my users asked me for some views that list the items in the database, a search tool, where I can type the name and a Job or Error for example and list only that information.

Is there any way to implement this today or some project for those who know how to add this functionality in the future?

**Comment by glutamate** [comment]
Yes you should:

1. build a filter view for this table where you include a search bar
2. build a page where you embed both the filter view and your list

does this solve your problem?

**Comment by Tacioandrade** [comment]
Thanks a lot, it worked! I didn't know how to correctly use the filter, I thought it was to generate a view with the data and it would show the table as a whole, but doing it this way worked!

Just a strange thing, after selecting the item I want in the combo box, the page reloads, filtering the data, but the combo box comes back empty, instead of with the Job Name value selected for example.

But this is just something aesthetic, the functionality is perfect!

Thanks!

**Comment by taewookim** [comment]
@glutamate I tried doing that but im having a hard time figuring it out for searching on tables when one of the columns has one-to-many relationship to another table.

Example: customer has many locations. Customer table has name and a column "location" that references "Location" table.  I want to search on both name AND location. So I built a list view on "Location" table, and in the view, I use JoinField to show the customer name along with the location address. When I add filter for the "Location" table, it searches neither location nor customer.

Is there a workaround?
